tests/test_runs/test_code_from_markdown.py

```diff
@@ -27,6 +27,11 @@ def find_code_block(file: Path, start_indicator: str, end_indicator: str) -> str
         ("extractor/val.md", "[comment]:vanilla-validation-start\n", "[comment]:vanilla-validation-end\n"),
         ("extractor/train_val_pl.md", "[comment]:lightning-start\n", "[comment]:lightning-end\n"),
         ("extractor/train_val_pl_ddp.md", "[comment]:lightning-ddp-start\n", "[comment]:lightning-ddp-end\n"),
+        (
```

Comment:
can you make it one line or it's black? if so, you can turn it off for this particular file

Reply:
Yeah, it's black. Okay, i'll add ignore for this line.